packages/editor/src/components/post-card-panel/index.js

```diff
@@ -25,89 +18,51 @@ import { store as editorStore } from '../../store';
-			const _type = getCurrentPostType();
-			const _id = getCurrentPostId();
-			const _record = getEditedEntityRecord( 'postType', _type, _id );
-			const _templateInfo =
-				[ TEMPLATE_POST_TYPE, TEMPLATE_PART_POST_TYPE ].includes(
-					_type
-				) && __experimentalGetTemplateInfo( _record );
-			return {
-				title:
-					_templateInfo?.title || getEditedPostAttribute( 'title' ),
-				id: _id,
-				postType: _type,
-				icon: unlock( select( editorStore ) ).getPostIcon( _type, {
-					area: _record?.area,
-				} ),
-				// Post excerpt panel and Last Edited info are rendered in different place depending on the post type.
-				// So we cannot make this check inside the PostExcerpt or PostLastEditedPanel component based on the current edited entity.
-				showPostContentPanels: [
-					TEMPLATE_POST_TYPE,
-					TEMPLATE_PART_POST_TYPE,
-					PATTERN_POST_TYPE,
-				].includes( _type ),
-			};
-		},
-		[]
-	);
+export default function PostCardPanel( { actions } ) {
+	const { title, icon } = useSelect( ( select ) => {
+		const {
+			getEditedPostAttribute,
+			getCurrentPostType,
+			getCurrentPostId,
+			__experimentalGetTemplateInfo,
+		} = select( editorStore );
+		const { getEditedEntityRecord } = select( coreStore );
+		const _type = getCurrentPostType();
+		const _id = getCurrentPostId();
+		const _record = getEditedEntityRecord( 'postType', _type, _id );
+		const _templateInfo =
+			[ TEMPLATE_POST_TYPE, TEMPLATE_PART_POST_TYPE ].includes( _type ) &&
+			__experimentalGetTemplateInfo( _record );
+		return {
+			title: _templateInfo?.title || getEditedPostAttribute( 'title' ),
+			id: _id,
+			icon: unlock( select( editorStore ) ).getPostIcon( _type, {
+				area: _record?.area,
+			} ),
+		};
+	}, [] );
 	return (
-		<PanelBody>
-			<div
-				className={ clsx( 'editor-post-card-panel', className, {
-					'has-description': showPostContentPanels,
-				} ) }
+		<div className="editor-post-card-panel">
+			<HStack
+				spacing={ 2 }
+				className="editor-post-card-panel__header"
+				align="flex-start"
 			>
-				<HStack
-					spacing={ 2 }
-					className="editor-post-card-panel__header"
-					align="flex-start"
+				<Icon className="editor-post-card-panel__icon" icon={ icon } />
+				<Text
+					numberOfLines={ 2 }
+					truncate
+					className="editor-post-card-panel__title"
+					weight={ 500 }
+					as="h2"
 				>
-					<Icon
-						className="editor-post-card-panel__icon"
-						icon={ icon }
-					/>
-					<Text
-						numberOfLines={ 2 }
-						truncate
-						className="editor-post-card-panel__title"
-						weight={ 500 }
-						as="h2"
-					>
-						{ title ? decodeEntities( title ) : __( 'No Title' ) }
-					</Text>
-					{ actions }
-				</HStack>
-				<VStack className="editor-post-card-panel__content">
-					{ showPostContentPanels && (
-						<VStack
-							className="editor-post-card-panel__description"
-							spacing={ 2 }
-						>
-							<PrivatePostExcerptPanel />
```

Comment:
We're are we rendering this element?

Reply:
We're rendering it but in a consistent place (it was rendered in two different places for different post types)

Reply:
Can you link where? Sorry if I'm missing the obvious logic change, but keeping up with all the unify/redesign changes for the sidebar is a bit tricky.

Reply:
yes 

https://github.com/WordPress/gutenberg/blob/a1a0d173d6fe932c21fcc8e3ffdc35bcf2e9b081/packages/editor/src/components/sidebar/post-summary.js#L15

https://github.com/WordPress/gutenberg/blob/a1a0d173d6fe932c21fcc8e3ffdc35bcf2e9b081/packages/editor/src/components/sidebar/post-summary.js#L71

Reply:
Sorry for the confusion. This works as expected.

P.S. We need to add some doc block comments for `PrivatePostExpectPanel` and `PostExpectPanel` to make them easily distinguishable without needing to read the logic.

cc @ntsekouras